integration-tests/vault/src/test/java/io/quarkus/vault/VaultITCase.java

```diff
@@ -323,6 +328,21 @@ private void assertDynamicCredentials(String clientToken, VaultAuthenticationTyp
         assertEquals(vaultDynamicCredentials.leaseId, vaultRenewLease.leaseId);
     }
 
+    private void assertDynamicRabbitMQCredentials(String clientToken, VaultAuthenticationType authType) {
```

Comment:
we can probably merge `assertDynamicDbCredentials` and `assertDynamicRabbitMQCredentials`.
`creds` can be replaced by `DEFAULT_REQUEST_PATH`.
for consistency, I would say let's add a `RABBITMQ_MOUNT` in `CredentialsProviderConfig`, which should probably be renamed into `X_DEFAULT_MOUNT`.

Reply:
Done.